src/components/VmUserMessages/index.js

```diff
@@ -67,11 +67,12 @@ class VmUserMessages extends React.Component {
     }
 
     let idCounter = 0
+    const badgeElem = userMessages.get('records').size === 0 ? null : <span className='badge' id={`${idPrefix}-size`}>{userMessages.get('records').size}</span>
```

Comment:
Could you please
* use full word: `badgeElement`
* add line breaks before `?` and `:` with proper indentation?